config/locales/bg.yml

```diff
@@ -886,7 +886,7 @@ bg:
       h: 1ч
       d: 1д
       M: 1мес
-      y: 1г
+      1: 1г
```

Comment:
@Nikerabbit, this has happened before and it causes a bug for us. The key should be `y`, not `1`. Is there a way you guys can prevent this from happening in the future?